examples/mlx_metal_kernel_opt/qwen3_benchmark_suite.py

```
@@ -0,0 +1,978 @@
+
+            print(f"📊 Running {MEASUREMENT_RUNS} measurement runs...")
+
+            # Measurement runs
+            successful_results = []
+            for run_idx in range(MEASUREMENT_RUNS):
+                try:
+                    print(f"   Measurement run {run_idx+1}/{MEASUREMENT_RUNS}...")
+
+                    # Clear cache before each measurement run for consistency
+                    mx.clear_cache()
+                    initial_memory = mx.get_active_memory()
```

Ruff (0.11.9)

Local variable `initial_memory` is assigned to but never used

Remove assignment to unused variable `initial_memory`

(F841)